### ISSUE TITLE
Wrong dimensions of S in graph_to_mask

### ISSUE THREAD

**Issue by jwardbond** [SAMPL-Weizmann/DeepCut#3]
Hello!

The last line in `util.graph_to_mask` is

```python
S = torch.tensor(np.reshape(S, (S.shape[0] * S.shape[0],))).type(torch.LongTensor)
```

My understanding of this function is that you first transform S from an Nx1 tensor to an H_patches x W_patches tensor in order to create the mask, and then you do the reverse of this to return S. In this case should S not be reshaped using both the width and height of S? i.e.

```python
S = torch.tensor(np.reshape(S, (S.shape[0] * S.shape[1],))).type(torch.LongTensor)
```
(note that I am using `S.shape[1]`, rather than `S.shape[0]`)

If you don't do this, it seems like `util.graph_to_mask` will only work with square images.

**Comment by amitt1236** [comment]
Hi, The code resizes the input image to square format. This is done before passing the image into the transformer for feature extraction. the mask is then resized again to original image size  (in the same function ).
We manipulate a square image, but output an original dimension image. 
You can look on the hyperparameter 'res' for the exact resolution.  
Did you encountered any problems ? unwanted behavior ?

**Comment by jwardbond** [comment]
> Hi, The code resizes the input image to square format. This is done before passing the image into the transformer for feature extraction. the mask is then resized again to original image size (in the same function ). We manipulate a square image, but output an original dimension image. You can look on the hyperparameter 'res' for the exact resolution. Did you encountered any problems ? unwanted behavior ?

Thanks for responding :) . I realized this was my mistake, as I assumed that `util.load_data_img` took an `int` for `image_size`, when it fact it was intended to be `[int,int]`.

It actually still works if `img_size` is just an `int` ... it just fails in `util.graph_to_mask`~
